website/content/docs/configure-hcp-packer.mdx

```diff
@@ -0,0 +1,88 @@
+---
+description: |
+  Packer can be configured to publish artifact metadata for all completed builds to an active HCP Packer registry.
+  The configuration for HCP Packer is managed by environment variables for legacy JSON templates or via a hcp_packer_registry
+  block for HCL2 templates. 
+page_title: Configuring Packer for HCP Packer
+---
+
+# Configuring Packer for HCP Packer
+
+HCP Packer stores metadata about your Packer images so that you can track updates, use the most up-to-date base images, and deploy the most up-to-date downstream images. 
+For more details on How it Works see the [HashiCorp Cloud Platform Packer documentation](https://cloud.hashicorp.com/docs/packer#what-is-hcp-packer) 
+
+You can configure any Packer template to push metadata to the HCP Packer registry, but we recommend that you use HCL2 Packer templates. 
+The HCL2 support in Packer allows you to codify how Packer interacts with the registry and customize your image metadata. 
+
+Users wishing to use HCP Packer with JSON are encouraged to get started with the environment variables and eventually work to migrate from JSON to HCL2 configuration
+templates. 
+
+## HCP Packer Environment Variables
+
+HCP Packer configuration through environment variables is currently the only way to publish artifacts from a Packer JSON template. 
+
+The HCP Packer environment variables are made available for users who wish to publish their artifacts to an active HCP Packer registry
+without making any changes to their existing template. Making it an ideal solution for users looking to get started with an existing template; both JSON and HCL2.
+
+HCP Packer environment variables must be configured before executing a Packer build command. They work with all template types but are required for legacy JSON templates. 
+
+** Required Environment Variables**
```

Comment:
One thought on this is --> we have a really clear list of requirements in the HCP Packer docs. Should we replicate those here? For example, we talk about the specific version of Packer you need, etc. I worry that if we don't, users may try to configure their connection from this alone and possibly get stuck. 

This is partially why (refer to my comment on the PR) I think it could be more beneficial to fold this into the existing HCP Packer docs. Regardless, I think we need to let users know about all the requirements!
```suggestion
You must set the following environment variables to enable Packer to push metadata to the HCP Packer registry.
```